mlflow/projects/_project_spec.py

```diff
@@ -58,10 +58,21 @@ def load_project(directory):
                     """E.g.: '[["NEW_VAR", "new_value"], "VAR_TO_COPY_FROM_HOST"])""")
 
     # Validate config if conda_env parameter is present
-    conda_path = yaml_obj.get("conda_env")
-    if conda_path and docker_env:
-        raise ExecutionException("Project cannot contain both a docker and "
-                                 "conda environment.")
```

Comment:
I am not sure you should remove this part. I think the #CRITEO part below is what you intend to keep only in our fork ? So when you will do the PR upstream you should leave this part to the check that there is no conda + docker env is still done.

Reply:
No, the purpose of the `#CRITEO` comments is to provide guidance to people when doing merges from the mlflow repo. When this PR is submitted in the official mlflow repo, this comments should disappear.